docs/tools/cli/project.rst

```diff
@@ -280,3 +280,33 @@ Removes a user with the supplied email address from the project.
 ::
 
   avn project user-remove --project my-project alice@example.com
+
+
+Request project SBOM
+--------------------
+
+SBOM reports are generated per project and can be downloaded as long as the necessary permissions are set for the user/account.
+
+.. _avn_project_generate_sbom:
+
+``avn project generate-sbom``
+''''''''''''''''''''''
+
+Downloads the SBOM report for this project, the SBOM is saved in the target file supplied by the user.
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``--project``
+    - The project to fetch details for
+  * - ``--sbom-filepath``
+    - File name, including path and extension (CSV or SPDX), to use
+
+**Example:** Download the SBOM report for the project "my-project", and save it in a file in the current directory called ``sbom.csv``.
+
+::
+
+  avn project generate-sbom --project my-project --sbom-filepath sbom.csv
```

Comment:
```suggestion
Gets the SBOM report download link for this project, in the chosen output format (CSV or SPDX).

.. list-table::
  :header-rows: 1
  :align: left

  * - Parameter
    - Information
  * - ``--project``
    - The project to fetch details for
  * - ``--output``
    - Output extension (CSV or SPDX), to use

**Example:** Get the SBOM report download link for the project "my-project" in ``csv``.

::

  avn project generate-sbom --project my-project --output csv
```

Reply:
As we modified the `aiven-client` feature + the [temporal external documentation located here](https://aiven.slab.com/posts/sbom-customer-external-doc-rvdyhhsa), suggesting the same changes here too.